VHbbAnalysis/Heppy/python/VHGeneratorAnalyzer.py

```diff
@@ -154,7 +154,7 @@ def makeMCInfo(self, event):
         event.genallbquarks = []
         event.genwzquarks = []
         event.gentopquarks  = []
-        event.genallstatus2bhadrons = [ p for p in event.genParticles if p.status ==2 and hasBottom(p.pdgId()) ]
+        event.genallstatus2bhadrons = [ p for p in event.genParticles if p.status() ==2 and self.hasBottom(p.pdgId()) ]
```

Comment:
ah right!

On Sat, Oct 24, 2015 at 10:04 AM, perrozzi <notifications@github.com> wrote:

> In VHbbAnalysis/Heppy/python/VHGeneratorAnalyzer.py
> <https://github.com/vhbb/cmssw/pull/228#discussion_r42931764>:
>
> > @@ -154,7 +154,7 @@ def makeMCInfo(self, event):
> >          event.genallbquarks = []
> >          event.genwzquarks = []
> >          event.gentopquarks  = []
> > -        event.genallstatus2bhadrons = [ p for p in event.genParticles if p.status ==2 and hasBottom(p.pdgId()) ]
> > +        event.genallstatus2bhadrons = [ p for p in event.genParticles if p.status() ==2 and self.hasBottom(p.pdgId()) ]
>
> it was a series of errors: hasBottom without self was just wrong but was
> not complaining because it never got there due to the missing () in
> p.status .... now is fully tested and fixed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vhbb/cmssw/pull/228/files#r42931764>.
>